docs/guides/features/icons.md

```diff
@@ -1,28 +1,45 @@
+import Command from '@theme/Command'
+
 # Icons
 
-Tauri ships with a default iconset based on its logo. This is probably NOT what you want when you ship your application. To remedy this common situation, Tauri provides the `tauricon` command that will take an input file ("./app-icon.png" by default) and create all the icons needed for the various platforms:
+Tauri ships with a default iconset based on its logo. This is NOT what you want when you ship your application. To remedy this common situation, Tauri provides the `icon` command that will take an input file (`"./app-icon.png"` by default) and create all the icons needed for the various platforms:
```

Comment:
```suggestion
Tauri ships with a default iconset based on its logo. This is NOT what you want when you ship your application. To remedy this common situation, Tauri provides the `icon` command that will take an input file (`"./app-icon.png"` by default) and create all the icons needed for the various platforms.
```

Or list out those platforms here if you want it

Reply:
nah, just a typo